internal/output/gcs/gcs.go

```diff
@@ -95,6 +95,7 @@ func NewClient(addr string) (gcsClient *storage.Client, ctx context.Context, can
 	if addr != "" {
 		h, err = url.Parse(addr)
 		if err != nil {
+			cancel()
```

Comment:
I think rather than doing this, `NewClient` should accept a `context.Context` and the caller should have the responsibility to handle this.